siac/main.go

```diff
@@ -24,6 +25,13 @@ var (
 	renterShowHistory bool   // Show download history in addition to download queue.
 )
 
+// exit codes
+// inspired by sysexits.h
+const (
+	exitCodeGeneral = 1
+	exitCodeUsage   = 64
+)
```

Comment:
can you extend the comment here? It's not really enough, I don't want to go look up sysexits.h to understand why these numbers are chosen, especially if you are reserving the right to deviate in small ways (which I think is the smart thing to do).